src/Tracker/trackerData.tsx

```diff
@@ -503,4 +503,62 @@ export const analyticsEventsMapping = new Map<
     'screen_welcome',
     { eventName: 'WELCOME', properties: { event_type: 'screen' } },
   ],
+
+  [
+    'screen_face_face_video_capture_record_button_click',
+    {
+      eventName: 'FACE_VIDEO_CAPTURE_RECORD_BUTTON_CLICKED',
+      properties: { event_type: 'action' },
+    },
+  ],
+  [
+    'screen_face_face_video_capture_recording_next_click',
+    {
+      eventName: 'FACE_VIDEO_CAPTURE_RECORDING_NEXT_CLICKED',
+      properties: { event_type: 'action' },
+    },
+  ],
+  [
+    'screen_face_face_video_confirmation_video_play',
+    {
+      eventName: 'FACE_VIDEO_CONFIRMATION_VIDEO_PLAY',
```

Comment:
I would rename this to `FACE_VIDEO_CONFIRMATION_PLAY`, less verbose

Reply:
done